src/Logs.tsx

```diff
@@ -0,0 +1,264 @@
+import React, { useState, useContext, useEffect, ReactElement } from 'react';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import moment from 'moment';
+import { Button, Dropdown, Header, Table, Icon } from 'semantic-ui-react';
+import { NumericOnlyInput } from './components';
+import { LogStatsContext, ClickedDayContext } from './Context';
+import { addExerciseToDatabase, retrieveExerciseDatabase, addLogToDatabase, retrieveLogDatabase, removeLogFromDatabase } from './Database';
+import './index.css';
+
+function Logs(): ReactElement {
+  const clickedDay = useContext(ClickedDayContext);
+  const [currentTab, setcurrentTab] = useState(<div />);
+
+  function handleClickDay(date: Date) {
+    clickedDay.setDate(moment(date));
+    setcurrentTab(<div />);
```

Comment:
setCurrentTab instead of setcurrentTab

Also I don't think that using JSX for state variables is recommended, you should have values that you can run if's on and then display JSX elements

Reply:
fixed